src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -120,25 +120,35 @@ async def do_update(self, id, data):
             {'prefix': self._config.datastore_prefix}
         )
 
+        if old['type'] == 'SYSCTL' and old['var'] in self.__default_sysctl and (
+            old['var'] != new['var'] or old['type'] != new['type']
+        ):
+            default_value = self.__default_sysctl.pop(old['var'])
+            cp = await run(['sysctl', f'{old["var"]}={default_value}'], check=False, encoding='utf8')
+            if cp.returncode:
+                self.middleware.logger.error(
+                    'Failed to set sysctl %r -> %r : %s', old['var'], default_value, cp.stderr
+                )
```

Comment:
Isnt the etc plugin supposed to do that?

Reply:
We won't have this information in etc plugin - this is an edge case

Reply:
Ah, different type, thanks